sunpy/visualization/imageanimator.py

```diff
@@ -529,15 +527,15 @@ def __init__(self, data, image_axes=[-2, -1], axis_range=None, **kwargs):
         if ax != list(range(self.naxis)):
             raise ValueError("spatial_axes and slider_axes mismatch")
 
-        self.axis_range = self._sanitize_axis_range(axis_range, data)
+        self.axis_ranges = self._sanitize_axis_ranges(axis_ranges, data)
 
         # create data slice
         self.frame_slice = [slice(None)] * self.naxis
         for i in self.slider_axes:
             self.frame_slice[i] = 0
 
-        base_kwargs = {'slider_functions': [self._updateimage]*self.num_sliders,
-                       'slider_ranges': [self.axis_range[i] for i in self.slider_axes]}
+        base_kwargs = {'slider_functions': [self.update_plot] * self.num_sliders,
+                       'slider_ranges': [self.axis_ranges[i] for i in self.slider_axes]}
         base_kwargs.update(kwargs)
         BaseFuncAnimator.__init__(self, data, **base_kwargs)
```

Comment:
I did it that way because otherwise I get this error when initialising ImageAnimator or LineAnimator:
```TypeError: 'super' object is not callable```